coderunners/lang/cpp.Dockerfile

```diff
@@ -2,16 +2,19 @@ FROM public.ecr.aws/sam/build-python3.9:latest
 
 ## Initial setup
 RUN yum install -y gcc-c++
+
 RUN pip install --upgrade pip
 RUN pip install awslambdaric -t "${LAMBDA_TASK_ROOT}"
 
 # Install dependencies
-COPY coderunner.requirements.txt ./
-RUN pip install -r coderunner.requirements.txt -t "${LAMBDA_TASK_ROOT}"
+COPY requirements-coderunners.txt ./
```

Comment:
I think it would look better if we have `coderunners/requirements.txt` here as well.
It makes sense to keep global requirements only if the project is ran as a single entity. Currently, our project is only executed in separate chunks. Executing coderunners does not need to have `boto3` or other dependencies to be able to be executed, therefore it makes more sense to keep its requirements isolated in its own directory.